cmd/agent-js-worker/go.mod

```diff
@@ -114,4 +114,8 @@ require (
 	nhooyr.io/websocket v1.8.3 // indirect
 )
 
-replace github.com/trustbloc/agent-sdk => ../..
+replace (
+	github.com/piprate/json-gold => github.com/trustbloc/json-gold v0.0.0-20220927171002-9dccd8860d42
+	github.com/trustbloc/agent-sdk => ../..
+	github.com/xeipuuv/gojsonschema v1.2.0 => github.com/Abdulbois/gojsonschema v1.2.1
```

Comment:
@troyronda @rolsonquadras do we need to fork gojsonschema too

Reply:
The discussion with @Abdulbois was to test wallet e2e flow locally with local fork. Once e3e works, then fork to trustbloc and refer to it.

Reply:
@Abdulbois - is e2e flow works in wallet now ?

Reply:
There are also alternative go libraries for json schema.

Reply:
> Does the e2e flow works in wallet now ?
I have checked all functionalites of `https://wallet.trustbloc.local:8091/` service and there was no such `stack size` error.  What do you mean by e2e in wallet? Setting up did-exchange with something. If so how I can do it?)

Reply:
I have checked all functionalites of `https://wallet.trustbloc.local:8091/` service and there was no such `stack size` error.  What do you mean by e2e in wallet? Setting up did-exchange with something. If so how I can do it?)

Reply:
@Abdulbois Please run oidc e2e flow described in https://github.com/trustbloc/wallet/blob/main/test/ui-automation/test/specs/oidc-flow.js#L275-L359.